app/clients/service/twilio_client.py

```
@@ -0,0 +1,21 @@
+
+    def send_message(self, to: str, from_: str, body: str):
+        try:
+            message = self.client.messages.create(
+                to=to,
+                from_=from_,
+                body=body
+            )
+            return message.sid
+        except Exception as e:
+            logger.error(f"Failed to send message to {to}: {str(e)}")
+            raise RuntimeError(f"Failed to send message to {to}: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)